jujugui/static/gui/src/app/app.js

```diff
@@ -536,7 +528,11 @@ YUI.add('juju-gui', function(Y) {
         entityPromise = this._fetchEntityFromUserState(pathState.state.user);
       }
       this.controllerAPI = this.setUpControllerAPI(
-        controllerAPI, user, password, macaroons, entityPromise);
+        controllerAPI,
+        controller.user,
```

Comment:
I can't suss out why, but making this change is really throwing off the anonymousMode tests; given we want to release today and this is just ugly rather than wrong, I'm going to punt on it--we can clean this up more as we continue to clean up app.js and do the user class refactor branches.